Add changes. (#2511)
wso2/docs-is: en/docs/fragments/oauth-app-pkce.md

## OAuth/OpenID Connect configuration

> In the created service provider, follow the below provided steps.

Reviewer: ```suggestion
Make the following changes to the created service provider.
```
